src/main/fc/runtime_config.h

```diff
@@ -45,9 +46,10 @@ typedef enum {
     ARMING_DISABLED_CMS_MENU    = (1 << 8),
     ARMING_DISABLED_OSD_MENU    = (1 << 9),
     ARMING_DISABLED_BST         = (1 << 10),
+    ARMING_DISABLED_NOPREARM    = (1 << 11)
```

Comment:
Will do, I saw there is a conflict here as well.  I could pull in the recent changes and rebase if there aren't any other issues.